### ISSUE TITLE
Split *meta entities into separate files

### ISSUE THREAD

**Issue by JanVoracek** [versionpress/versionpress#883]
Now we save postmeta, usermeta, termmeta and commentmeta together with their "parent" entity. Maybe we should consider to save them individually. It will be better for merging, reverts, the code will be more simple and straightforward. 

The only disadvantages are:
- a lot of INI files
- maybe a worse context in a full diff – the user will see only the postmeta, not the post itself (or maybe it will be an advantage, who knows)

**Comment by borekb** [comment]
Technically, I think we should do this – it's kind of similar to the options split (#473). The diff could be improved on the UI, no problem there.

Agree with the milestone – we should release this as part of 3.0.

**Comment by JanVoracek** [comment]
Moving this to 4.0. Bug #876 is caused by wrong detection of PHP binary – there's no merge conflict. However, it's maybe right thing to do. We will discuss it again in 4.0.

**Comment by borekb** [comment]
Agree, it's not clear what the right approach is here. Let's watch the issue some more, if people report issues with the current storage format etc.

**Comment by borekb** [comment]
As some of the issues linking here (#1147, #1148) show, this should be done. The best milestone for it is 4.0-beta, let's see if we can manage to squeeze it there.
